media-tv/plex-media-server/plex-media-server-1.40.4.8679.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit readme.gentoo-r1 systemd unpacker pax-utils
+
+MY_PV="${PV}-424562606"
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+
+DESCRIPTION="Free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	arm? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_armhf.deb )
+	arm64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_arm64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )
+"
+S="${WORKDIR}"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm ~arm64 ~x86"
+RESTRICT="bindist"
+
+DEPEND="
+	acct-group/plex
+	acct-user/plex"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}.service.patch"
+)
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/plexmediaserver/lib/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so"
+)
+
+BINS_TO_PAX_MARK=(
```

Comment:
Currently there is the following QA warning:
  VariableScope: version 1.40.4.8679: variable 'ED' used in 'global scope' on lines: 44, 45

I think the easiest way to fix it would be to set BINS_TO... directly inside src_install phase

Thanks!

Reply:
Apparently a rebase clears it up. eli the wizard guy broke something and it broke all open PR's around this time. Good now!

Reply:
Sorry but I don't understand. Last commit still sets BINS_TO_PAX_MARK at global scope and, as a consequence, pkgdev QA tests warns as it doesn't like to rely on ${ED} at global scope
https://pkgcore.github.io/pkgcheck/man/pkgcheck.html#variablescope

Reply:
It's a warning I don't really care about though, I was confused as to why pkgcheck failed before. Previously pkgcheck was reporting as broken and now it's not.  I thought it was caused by this variable which I haven't changed between releases (I did not add the PAX_MARK stuff, some other dev did.... so as I don't really feel comfortable changing it unless there's complaints, I'm leaving it)